### ISSUE TITLE
Write assertions for focusable elements v. tabbable index

### ISSUE THREAD

**Issue by 1Copenut** [department-of-veterans-affairs/vets-website#7554]
These should be custom assertions in Nightwatch that can be shared across products, so follow the local coding style.
